--- lis/include/lis.h.org	2016-09-10 21:35:29.334624333 +0200
+++ lis/include/lis.h	2016-09-10 21:37:30.550783992 +0200
@@ -337,7 +337,12 @@
 #else
 typedef double LIS_REAL;
 #if defined(HAVE_COMPLEX_H)
+#ifdef __cplusplus
+#include <complex>
+typedef std::complex<double> LIS_COMPLEX;
+#else
 typedef double complex LIS_COMPLEX;
+#endif
 #else
 #if defined(_Complex_I) && defined(complex) && defined(I)
 typedef double _Complex LIS_COMPLEX;
